client/app/index.html

```diff
@@ -221,6 +222,8 @@ <h3 translate>About</h3>
             'liveblog.posts',
             'liveblog.blog',
             'liveblog.themes',
+            'liveblog.freetypes',
```

Comment:
one module is to manage freetypes CRUD operation 
and one is for managing the freetype manipulation in blog edit.